pkg/device/device-util.go

```diff
@@ -211,8 +211,8 @@ func RunCommand(cList []string) (string, error) {
 }
 
 // GetDiskList Todo
-func GetDiskList() ([]string, error) {
-	var result []string
+func GetDiskList() ([]apis.Device, error) {
```

Comment:
@praveengt Have changed this method signature as we have a struct for the `Device` now which is introduced as part of the capacity tracking changes.